component/http/errors.go

```diff
@@ -16,65 +16,65 @@ func (e *Error) Error() string {
 	if e.payload == nil {
 		return fmt.Sprintf("HTTP error with code: %d", e.code)
 	}
-	return fmt.Sprintf("HTTP error with code: %d payload: %v", e.code, e.payload)
+	return fmt.Sprintf("HTTP error with code: %d Payload: %v", e.code, e.payload)
```

Comment:
any reason for this change and all the other changes? we follow a lower case rule in go when it comes to errors...

Reply:
nope, it was a bulk replace side-effect